### DISCUSSION TITLE
Hover Plugin: missing formatTimeCallback ?

### DISCUSSION THREAD

**Discussion by Adrii77** [katspaugh/wavesurfer.js#3626] [Q&A] [answered]
Hello,

Just started to migrate from 6 to 7.
I'm using Cursor plugin, which became Hover as the doc said. 

Problem : With Cursor, I'm using the `formatTimeCallback` function (to have a hh:mm:ss format) and it seems to not be included in Hover. [I saw a similar question about Timeline plugin](https://github.com/katspaugh/wavesurfer.js/discussions/2985).

Is it intentional ?

Thanks !

**Comment by katspaugh** [comment, marked as answer]
Same answer as in the timeline thread, `formatTimeCallback` was unfortunately not ported. It should be fairly easy to add it.

**Comment by Adrii77** [comment]
OK, I'll create a pull request with this option.

Shall I create an issue from this discussion ? (Sorry, I've never contribute to an open source project yet)

**Reply by katspaugh** [reply]
Awesome! You can just link your PR to this thread.

**Reply by Adrii77** [reply]
Here it is : https://github.com/katspaugh/wavesurfer.js/pull/3628
